pkgs/games/clonehero/fhs-wrapper.nix

```diff
@@ -0,0 +1,39 @@
+{ clonehero-unwrapped
+, clonehero-xdg-wrapper
```

Comment:
This doesn't exist anymore, so this package doesn't build.

Reply:
Oh thank you! Sorry about that. I just submitted a fix and everything is working like it was before I removed `clonehero-xdg-wrapper` from `all-packages.nix`